docs/dashboards/custom-component.md

```diff
@@ -77,4 +79,66 @@ Dashboards.board({
     }]
 });
 ```
-[The live example can be found here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboards/demos/custom-component).
\ No newline at end of file
+[The live example can be found here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboards/demos/custom-component).
+
+---
+## Custom HTML Component
+The basic HTML component described in the [Types of Components](https://www.highcharts.com/docs/chart-and-series-types/dashboards/types-of-components) page is very flexible, but requires a lot of configuration. In this example, we will create a custom HTML component, which will be more flexible and easier to use.
+
+This custom component will be extended from the basic HTML component, so we will need to import the `HTMLComponent` class. The easiest way to do so is through the `ComponentRegistry` as shown below. We also will use the `Highcharts.AST` class, which will be used to parse the string type HTML into the AST-like object.
+
+Then we can create our custom class, which will inherit from the `HTMLComponent` class. The only thing we need to do is to create a method to extract the HTML from the options and parse it into the AST-like object. In the example this method is called `getCustomHTML` and it assigns the element generated by the AST to the `elements` property. The only thing left is to register the new component in the `ComponentRegistry` and we are ready to use it.
+
+```js
+const { ComponentRegistry } = Dashboards,
+    HTMLComponent = ComponentRegistry.getComponent('HTML'),
+    AST = Highcharts.AST;
+
+class CustomHTML extends HTMLComponent {
+    constructor(cell, options) {
+        super(cell, options);
+        this.type = 'CustomHTML';
+        this.getCustomHTML();
+        return this;
+    }
+
+    getCustomHTML() {
+        const options = this.options;
+        if (options.id) {
+            const customHTML = document.getElementById(options.id).outerHTML;
+
+            this.options.elements = new AST(`${customHTML}`).nodes;
+        } else if (options.html) {
+            this.options.elements = new AST(options.html).nodes;
+        }
+    }
+}
+
+ComponentRegistry.registerComponent(CustomHTML);
+```
+
+This custom component can generate HTML elements to you dashboards by:
+- referencing the HTML element by its `id` attribute
+- passing the HTML as a string to the `html` property
+
+Use of this component is shown below:
```

Comment:
Maybe, to be consistent across the docs, both YouTube and CustomHTML components should have the same structure and text?